src/controller/AutoCommissioner.cpp

```diff
@@ -258,10 +260,22 @@ CHIP_ERROR AutoCommissioner::StartCommissioning(DeviceCommissioner * commissione
 
 Optional<System::Clock::Timeout> AutoCommissioner::GetCommandTimeout(CommissioningStage stage)
```

Comment:
It looks like there is no longer any intent for this to return an optional (always returns a value), so could remove the Optional<> in the return value.

Reply:
This returns an optional because the DeviceController function takes an optional. The auto commissioner always passes in a timeout value, but it's not required by the API. If the optional is NullOptional, the default is used. So this will need to be turned into an optional at some point - I opted to do it in the function return so I could use the return value directly.